server/routes/proxyLinks.js

```diff
@@ -6,33 +6,33 @@ import bodyParser from 'body-parser';
 app.use(cors());
 app.use(bodyParser.json());
 import { minioClient } from '../minioConfig.js';
-// import { get_user_bucket } from '../Database_queries/queries.js';
+import { logger, log } from '../logger.js';
 
 router.get('/thumbnail/:url', async function (req, res) {
   try {
     const user = req.params.url;
     let bucketName = 'datadrive-dev';
-    const { imageName, fileId } = req.query;
+    const { imageName } = req.query;
     minioClient.getObject(
       bucketName,
-      'hv/' + user + '/thumbnail/' + imageName + '.png' + fileId,
+      'hv/' + user + '/thumbnail/' + imageName + '.png',
       (err, dataStream) => {
         if (err) {
-          console.error('Error getting object from Minio:', err);
+          log.error('Error getting object from Minio:', err);
           res.status(500).send('Internal Server Error');
           return;
         }
 
         dataStream.on('error', readErr => {
-          console.error('Error reading Minio object:', readErr);
+          log.error('Error reading Minio object:', readErr);
```

Comment:
Similar to the previous comment, include more context in the error log for reading the Minio object. This will help in debugging issues related to specific objects.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*